content/applications/general/users/facebook.rst

```diff
@@ -0,0 +1,121 @@
+===============================
+Facebook sign-in authentication
+===============================
+
+The *Facebook* OAuth sign-in function allows Odoo users to sign in to their database with their
+Facebook account.
+
+.. warning::
+   Databases housed on Odoo.com should **not** use OAuth login for the owner or administrator of the
+   database, as it would unlink the database from their Odoo.com account. If OAuth is setup for
+   that user, the database can no longer be duplicated, renamed, or otherwise managed from the
+   Odoo.com portal.
+
+Meta for Developers setup
+=========================
+
+Go to `Meta for Developers <https://developers.facebook.com/>`_ and log in. Click :guilabel:`My
+Apps`. On the :guilabel:`Apps` page, click :guilabel:`Create App`.
+
+On the :guilabel:`Use cases` page, select :guilabel:`Authenticate and request data from users with
+Facebook Login`, the click :guilabel:`Next`.
+
+In the :guilabel:`Add an app name` field, enter `Odoo Login OAuth`, or a similar title.
+
+.. note::
+   The :guilabel:`App contact email` automatically defaults to the email address associated with the
+   Meta account. If this email address is not regularly monitored, it may be wise to use another
+   email address.
+
+Click :guilabel:`Next`. Review the :guilabel:`Publishing requirements`, the :guilabel:`Meta
+Platform Terms` and :guilabel:`Developer Policies`. Then, click :guilabel:`Create app`.
+
+.. important::
+   Clicking :guilabel:`Create app` may require password re-entry.
+
+Customize app
+-------------
+
+After the new app is created, the :guilabel:`Dashboard` page appears, with a list of steps to be
+completed before the app can be published. From here, click :guilabel:`Customize adding a Facebook
+Login button`.
+
+.. image:: facebook/app-requirements.png
+   :align: center
+   :alt: The App Dashboard in the Meta for developers platform.
+
+On the :guilabel:`Customize` page, click :guilabel:`Settings`.
+
+In the :guilabel:`Valid OAuth Redirect URIs` field, enter `https://<odoo base
+url>/auth_oauth/signin`, replacing `<odoo base url>` with the URL of the applicable database. Click
```

Comment:
What's a base url? Is it just up to odoo.com?

For example:
https://65632148-17-0-all.runbot207.odoo.com/ ?

Reply:
correct, I will add an example to make this clearer